### DISCUSSION TITLE
B站开启了首页的新一轮内测

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3285] [综合讨论 / General]
目前还是灰度测试，跟现在的新版首页大同小异，有一种脚本又要失效的预感...

V1残党大失败（悲）

**Comment by the1812** [comment]
长啥样(

**Reply by q56535** [reply]
![image](https://user-images.githubusercontent.com/26663837/166102675-b8cf872d-1ece-480d-813d-89177486981f.png)
![image](https://user-images.githubusercontent.com/26663837/166102690-62b9f85a-c791-4fa2-8669-041c575cd8e1.png)
经测试，首页已改成无限滚动式

**Reply by Geeyun-JY3** [reply]
这内测的首页无限滚动又学的 YouTube 是吧
